### ISSUE TITLE
Maybe add a default bottom padding to line charts

### ISSUE THREAD

**Issue by wangqianwen0418** [gosling-lang/gosling.js#122]
Maybe we can consider add a default bottom padding to line charts.

Otherwise, the line is almost invisible when the y value is small (e.g., sample 4 in the example below).
![image](https://user-images.githubusercontent.com/19774198/105651438-e8d68900-5e84-11eb-9396-eeca7782635e.png)

**Comment by sehilyi** [comment]
I agree, I think an easy fix would be to add a padding to the `range` of `y` scale internally:

https://github.com/gosling-lang/gosling.js/blob/43626eaf21417bf36128a405dceeaa6ee00d0851/src/core/gosling-track-model.ts#L575

**Comment by etowahadams** [comment]
I tried Sehi's suggestion to add padding to the y-scale. This works, but it also moves the position of the zero such that it is not fully visible in the bottom line. 

I've been trying a couple of different alternative strategies and will report back. This requires a deeper understanding of how the axis works, so I've been spending most of my time building this understanding. 

<img width="289" alt="image" src="https://user-images.githubusercontent.com/14843470/230418546-d9bf15bc-653e-41b4-9b60-deeac2cc90a7.png">

**Comment by sehilyi** [comment]
As part of a post-1.0 cleanup of Gosling issues, we are closing stale issues that are no longer planned. If this is still relevant, feel free to reopen or start a discussion. Thanks!
